src/lib/hooks/use-fetch-modal-jobs.js

```diff
@@ -0,0 +1,12 @@
+import {useEffect} from "react";
+import {useRecoilValue, useSetRecoilState} from "recoil";
+import {careerJobsState, jobsState} from "lib/recoil";
+
+export default function useFetchModalJobs() {
```

Comment:
I think this should just be done with the selectors. I don't think you need a hook to activate it

Reply:
Great catch! I've removed all of the job hooks and am now using the selector!